EasyVCR/Censors.cs

```diff
@@ -238,16 +243,16 @@ public static string CensorJsonData(string data, string censorText, IReadOnlyCol
             {
                 var jsonDictionary = JsonSerialization.ConvertJsonToObject<Dictionary<string, object>>(data);
                 var censoredJsonDictionary = ApplyDataCensors(jsonDictionary, censorText, elementsToCensors);
-                return censoredJsonDictionary == null ? data : JsonSerialization.ConvertObjectToJson(censoredJsonDictionary);
+                return JsonSerialization.ConvertObjectToJson(censoredJsonDictionary);
             }
             catch (Exception)
             {
-                // body is not a JSON dictionary
+                // body is not a JSON dictionary 
```

Comment:
Nit: The trailing space seems counterintuitive?

Reply:
Resolved